src/frontend.scss

```diff
@@ -4,3 +4,8 @@ button:focus {
 	box-shadow: 0 0 0 2px rgba(0, 125, 250, 0.8);
 	border-radius: 2px;
 }
+
+@font-face {
+	font-family: 'Noto Sans';
+	src: url('../assets/fonts/NotoSans-VariableFont_wdth-wght.ttf');
```

Comment:
I remember there being some confusion about the fonts. I thought we were using the system default font stack, and Notowas just what Ivan was using for designs… Or did we confirm that we do need to ship this font? Sorry if I missed some communication about this

Reply:
Confirmed on the call yesterday that Noto Sans is correct :)